resources/settings.xml

```diff
@@ -10,6 +10,7 @@
         <setting label="30904" type="text" id="warning" enable="false"/> <!-- disabled_warning -->
         <setting label="30905" help="30906" type="slider" id="update_frequency" default="31" range="1,3,90" option="int" enable="eq(-2,false)" visible="!system.platform.android"/>
         <setting label="30907" help="30908" type="folder" id="temp_path" source="" option="writeable" default="special://masterprofile/addon_data/script.module.inputstreamhelper" visible="!system.platform.android"/>
+        <setting label="Auto-install Widevine CDM" type="bool" id="automatic_install" default="false"/>
```

Comment:
I think "Unattended Widevine CDM install" is a better description.

Reply:
Agreed.